src/components/home/homeTopSection/homeTopSection.tsx

```diff
@@ -1,108 +1,39 @@
-    } else {
-      facetList = [];
-      dsCount = 0;
-    }
-  }
-
   return (
     <Col className="px-2">
-      <Row
-        className="w-100 bg-primary rounded mx-0 mb-3 pt-5 pb-5 pe-4 text-white"
-        style={{
-          backgroundImage: `url(${bundeslaender})`,
-          backgroundRepeat: "no-repeat",
-          backgroundSize: "450px",
-          backgroundPosition: "left 50px bottom -245px",
-        }}
-      >
-        <Col className="col-3">
-          <Row className="ps-4 mb-4">
-            <h3 className="fw-bold">GHGA Metadata Catalog</h3>
-          </Row>
-          <Row>
-            <div
-              className="border border-black border-2 p-0 ms-3 py-5 px-5 fw-bold text-center"
-              style={{ background: "RGBA(0,0,0,0.3)", borderRadius: "15px" }}
-            >
-              <p className="mb-4">
-                A FAIR Portal for Human Genomics data with GDPR-compliant access
-                control.
-              </p>
-              <NavLink to="/browse">
-                <Button
-                  variant="white"
-                  className="py-2 mb-2 text-secondary fw-bold shadow-md-dark border-secondary"
-                >
-                  <span className="text-uppercase">Get started</span>
-                </Button>
-              </NavLink>
-            </div>
-          </Row>
+      <Row className="w-100 mx-0 mb-3 pb-5 pe-4">
+        <Col className="col-auto ps-5">
+          <img src={germany} alt="Germany" height="450px" />
         </Col>
-        <Col>
-          <Row className="justify-content-center">
-            <HomeSearchbar filterDict={filterDict} />
-          </Row>
-          <Row className="mb-4 mt-2 justify-content-center">
-            <HomeFilterSelects
-              filterDict={filterDict}
-              facetList={facetList}
-              setFilterDict={setFilterDict}
-            />
-          </Row>
-          <Row className="mb-4 justify-content-center">
-            <Container className="col-3 text-center">
-              <NavLink to="/browse">
-                <Button
-                  variant="white"
-                  className="shadow-md-dark fs-5 fw-bold px-5"
-                >
-                  {dsCount}&nbsp;Total&nbsp;Datasets
-                </Button>
-              </NavLink>
-            </Container>
-          </Row>
-          <Row className="text-black justify-content-center">
-            <TopSectionBadges summaryStats={summary} />
-          </Row>
+        <Col className="p-5">
+          <p className="fw-bold text-center fs-4 px-5 mx-5 mb-4">
+            A FAIR Portal for Human Genomics Data with GDPR compliant Access
+            Control
+          </p>
+          <p className="mb-5 fs-5 px-5 mx-5">
+            By providing a secure home for their data, GHGA enables data
+            controllers to share data safely. GHGA further facilitates discovery
+            and supports the scientific community in maximising the value of
```

Comment:
Could you make different colors with a radius for the FAIR Principle ? check design

Reply:
Tried, but gradient texts don't work. Left it just as secondary